Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.10-alpine
+
+COPY requirements.txt ./requirements.txt
+COPY tgmusicbot.py ./tgmusicbot.py
+
+RUN python -m pip install -r requirements.txt
+RUN rm -rf /root/.cache/pip
+
+ENTRYPOINT [ "python", "tgmusicbot.py" ]
```

Comment:
text files should have new line characters at the end of the file

Reply:
docker-compose.yml has the same issue

Reply:
fixed